CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing to ROCm Docs
+
+The value of documentation being open-source is that it is open for contribution to all. We encourage everyone to contribute: if something was unclear to you, it may be unclear to others too. If you know some piece of information made sense being mentioned at a given point, add a link or excerpt. Don't know how to address a shortcoming you sense: open an issue. Community-driven documentation will surely tailor to a wider audience than those written centrally.
+
+Be polite and respectful in your contributions, content or otherwise. Authors, maintainers of these docs act on good intentions and to the best of their knowledge. Keep that in mind while you engage. Should you have issues with contributing itself, refer to [discussions](https://github.com/RadeonOpenCompute/ROCm/discussions) on the GitHub repository.
```

Comment:
We need community guidelines. I will work to set community guidelines. This section can stay until we adopt community guidelines.